validator/src/main.rs

```diff
@@ -2252,11 +2261,15 @@ pub fn main() {
             .ok()
             .map(|mb| mb * MB);
 
-    let filler_account_count = value_t!(matches, "accounts_filler_count", usize).ok();
+    let filler_accounts_config = FillerAccountsConfig {
+        count: value_t!(matches, "accounts_filler_count", usize).unwrap(),
+        size: value_t!(matches, "accounts_filler_size", usize).unwrap(),
```

Comment:
```suggestion
        count: value_t_or_exit!(matches, "accounts_filler_count", usize),
        size: value_t_or_exit!(matches, "accounts_filler_size", usize),
```

nicer than an unwrap panic